apps/frontend/src/pages/root/components/circular-progress/circular-progress.tsx

```diff
@@ -0,0 +1,65 @@
+import { useEffect, useState } from "~/libs/hooks/hooks.js";
+
+import styles from "./styles.module.css";
+
+const ZERO = 0;
+const HUNDRED = 100;
+const TWO = 2;
+const SIXTY = 60;
+
+const CircularProgress: React.FC = () => {
+	const [animatedValue, setAnimatedValue] = useState<number>(ZERO);
+	const radius = 43;
```

Comment:
```suggestion
	const RADIUS = 43;
```
Define it outside of component